src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -100,8 +101,9 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
       case 'days7':
       case 'days30':
       case 'year1':
-        return toPercentString(data / 100)
+        return data == null ? 'N/A' : toPercentString((data as number) / 100)
```

Comment:
If data is a number, need to change to `Number(data)`. Casting isn't sufficient.